tests/CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ function(ly_add_utest)
     endforeach()
 
     add_executable(${TEST_NAME} ${TEST_SOURCES} $<TARGET_OBJECTS:yangobj>)
+    target_compile_definitions(${TEST_NAME} PRIVATE LIBYANG_BUILD)
```

Comment:
Comment for the commit adding this (saying that `yangobj` target should be removed altogether): I totally agree but the reason for it being written this way is that there are test for some functions in the internal *libyang* API. I was against this from the beginning (it was not my call then) and the latest plans were to fix this and rewrite the tests to use only public API but we did not make this change yet.

Reply:
that makes sens,e I've adjusted the commit message.